examples/exampleLoaderTemplate_pyqt5.py

```diff
@@ -1,10 +1,11 @@
 # -*- coding: utf-8 -*-
 
-# Form implementation generated from reading ui file 'exampleLoaderTemplate.ui'
+# Form implementation generated from reading ui file 'examples/exampleLoaderTemplate.ui'
 #
-# Created by: PyQt5 UI code generator 5.12.3
+# Created by: PyQt5 UI code generator 5.15.4
```

Comment:
Hmm...this _might_ be an issue, I'll investigate if this is backward compatible.